controls/def.cf

```diff
@@ -667,7 +667,7 @@ bundle common def
         expression => some( ".+", "def.control_executor_runagent_socket_allow_users" );
 
       "cfengine_recommendations_enabled"
-        expression => "!cfengine_recommendations_disabled";
+        expression => "!cfengine_recommendations_disabled.!data:cfengine_recommendations_disabled";
```

Comment:
I am not a big fan of having multiple classes that do the same thing generally. This class is suggested because it's not clear how to define the class in MP CMDB UI, seems to me better to improve this elsewhere than add another class.

Reply:
Yes, I completely agree. This is really just my "user story" persona trying to scratch my own itch.